src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -75,6 +77,16 @@
     this.format = format;
   }
 
+  /**
+   * Informs if the element has an "id" attribute.
```

Comment:
I would also add a line explaining that this is done because we maintain unique Ids for all elements that have it